src/javascript/app/base/header.js

```diff
@@ -60,8 +61,38 @@ const Header = (() => {
         fullscreen_map.event.forEach(event => {
             document.addEventListener(event, onFullScreen, false);
         });
+        applyFeatureFlags();
     };
-    
+
+    const applyFeatureFlags = () => {
+        getRemoteConfig(true)
+            .then(data => {
+                const { cs_chat_livechat, cs_chat_whatsapp } = data.data;
+                const mobile_menu_livechat                   = getElementById('mobile__menu-livechat');
+                const livechat                               = getElementById('livechat');
+                const topbar_whatsapp                        = getElementById('topbar-whatsapp');
+                const whatsapp_mobile_drawer                 = getElementById('whatsapp-mobile-drawer');
+
+                if (cs_chat_livechat) {
```

Comment:
I think this part can be improved, as the logic with applying inline-flex and none is similar.
 Something like this:

```
 const updateVisibility = ({condition, element_1, element_2}) => {
    const display_type= condition ? 'inline-flex' : 'none';
    element_1.style.display = display_type;
    element_2.style.display = display_type;
}

updateVisibility({condition: cs_chat_livechat, element_1: mobile_menu_livechat, element_2: livechat});
updateVisibility({condition: cs_chat_whatsapp, element_1: topbar_whatsapp, element_2: whatsapp_mobile_drawer});
```